src/TimeSeries.jl

```diff
@@ -9,8 +9,8 @@ using Base.Dates
 export TimeArray, AbstractTimeSeries,
        by, from, to, findwhen, findall, timestamp, values, colnames, meta,
        lag, lead, percentchange, moving, upto,
-       basecall,
-       merge, collapse,
+       basecall, uniformlyspaced, uniformlyspace,
+       merge, merge_left, collapse,
```

Comment:
Can we keep only `merge` and make `merge_left` available with a kwarg, setting it to false?